File: src/main/java/br/gov/pf/model/entity/UserRole.java
package br.gov.pf.model.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;
import javax.validation.constraints.NotNull;

@Entity
@Table(name = "users_roles")
public class UserRole extends AbstractEntity {

    @Column(name = "description")
    private @NotNull String description;

    public UserRole(@NotNull String description) {
        this.description = description;
    }

    public UserRole() {
    }

    /**
     * GETS E SETS
     */
    public String getDescription() {
        return this.description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

}

File: src/main/java/br/gov/pf/model/entity/UserRole.kt
package br.gov.pf.model.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Table;
import javax.validation.constraints.NotNull;

@Entity
@Table(name = "users_roles")
class UserRole : AbstractEntity() {

    @Column(name = "description")
    @NotNull
    var description: String = ""

}
